lib/Utils.js

```diff
@@ -49,14 +49,15 @@ function unserializeUids(stringdata) {
 */
 function objectCopy(target) {
     const result = target;
-    let source;
-    function copy(f) {
-        result[f] = source[f];
-    }
     /* eslint-disable prefer-rest-params */
-    for (let i = 1; i < arguments.length; i++) {
-        source = arguments[i];
-        Object.keys(source).forEach(copy);
+    const nb = arguments.length;
+    for (let i = 1; i < nb; i++) {
+        const source = arguments[i];
+        for (const key in source) {
+            if (source[key]) {
```

Comment:
I don't know if it's relevant, but if `source[key]` is false-ish you are not going to copy it.